src/Client/components/ques/written-ques.tsx

```
@@ -1,143 +1,200 @@
-                    onChange={(e: { target: { checked: boolean } }) => handleLongAnswerChange(question.id, e.target.checked)}
-                    isDisabled={question.isShortAnswer}
-                  >
-                    Long Answer
-                  </Checkbox>
-                </label>
+                <Input
+                  className="w-32"
+                  type="number"
+                  label="Points"
+                  value={question.points.toString()}
+                  onChange={(e) => handlePoints(question.id, parseInt(e.target.value))}
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)